### ISSUE TITLE
Make the app responsive

### ISSUE THREAD

**Issue by itaditya** [itaditya/JsImgEditor#11]
Currently the web app is not optimized to run on mobile. Any PR to remedy that is welcome

**Comment by VijayGomathi** [comment]
I will change it

**Comment by VijayGomathi** [comment]
sorry I can't

**Comment by 199911** [comment]
I can help tuning the font size and input size

**Comment by bobharley** [comment]
I would like to help on this

**Comment by itaditya** [comment]
Also guys, if you could star the repo it would be very beneficial for me

**Comment by felipemfarias** [comment]
Can I help with this?

**Comment by itaditya** [comment]
@felipemfarias sure go ahead

**Comment by lostdesign** [comment]
A year later, is this still wanted ^^?

**Comment by itaditya** [comment]
@lostdesign yes it is still required. Do let me know if you'd like to work on this. I'd provide all the help I can.

**Comment by rajat2502** [comment]
@itaditya is this still open to contribute sir?

**Comment by itaditya** [comment]
Yes it is still open @rajat2502. I'd suggest making some rough mockups either by hand or by some software and getting it approved by me before writing any code. Although you should take a look at the current code too so you get an idea what's possible in the time you can allocate.

**Comment by ashimasingla20** [comment]
Hi i would like to work on this.

**Comment by itaditya** [comment]
Go ahead!
